examples/imagenet/train_imagenet_data_parallel.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+"""Example code of learning a large scale convnet from ILSVRC2012 dataset.
```

Comment:
Please explain that this code uses MultiprocessParallelUpdater, and thus this code requires the installation of NCCL.